internal/host/transition_test.go

```diff
@@ -1507,6 +1509,176 @@ var _ = Describe("Refresh Host", func() {
+				statusInfoMsg: "Insufficient CPU to deploy CNV. Required CPU count is: 2 but found: 1",
+				validationsChecker: makeJsonChecker(map[validationID]validationCheckResult{
+					IsConnected:                 {status: ValidationSuccess, messagePattern: "Host is connected"},
+					IsHostnameValid:             {status: ValidationSuccess, messagePattern: ""},
+					IsAPIVipConnected:           {status: ValidationSuccess, messagePattern: "API VIP connectivity success"},
+					BelongsToMajorityGroup:      {status: ValidationPending, messagePattern: "Machine Network CIDR or Connectivity Majority Groups missing"},
+					HasInventory:                {status: ValidationSuccess, messagePattern: "Valid inventory exists for the host"},
+					HasMinCPUCores:              {status: ValidationFailure, messagePattern: "The host is not eligible to participate in Openshift Cluster because the minimum required CPU cores for any role is 2, found only 1"},
+					HasMinMemory:                {status: ValidationSuccess, messagePattern: "Sufficient minimum RAM"},
+					HasMinValidDisks:            {status: ValidationSuccess, messagePattern: "Sufficient disk capacity"},
+					IsMachineCidrDefined:        {status: ValidationSuccess, messagePattern: "Machine Network CIDR is defined"},
+					HasCPUCoresForRole:          {status: ValidationFailure, messagePattern: "Require at least 2 CPU cores for worker role, found only 1"},
+					HasMemoryForRole:            {status: ValidationSuccess, messagePattern: "Sufficient RAM for role worker"},
+					IsHostnameUnique:            {status: ValidationSuccess, messagePattern: "Hostname worker is unique in cluster"},
+					BelongsToMachineCidr:        {status: ValidationSuccess, messagePattern: "Host belongs to machine network CIDR 1.2.3.0/24"},
+					IsPlatformValid:             {status: ValidationSuccess, messagePattern: "Platform RHEL is allowed"},
+					IsNTPSynced:                 {status: ValidationFailure, messagePattern: "Host couldn't synchronize with any NTP server"},
+					AreContainerImagesAvailable: {status: ValidationPending, messagePattern: "Missing container images statuses"},
+					AreLsoRequirementsSatisfied: {status: ValidationSuccess, messagePattern: ""},
+					AreOcsRequirementsSatisfied: {status: ValidationSuccess, messagePattern: "ocs is disabled"},
+					AreCnvRequirementsSatisfied: {status: ValidationFailure, messagePattern: "Insufficient CPU to deploy CNV. Required CPU count is: 2 but found: 1"},
+				}),
+			},
+			{
+				name:          "insufficient master cpu",
+				hostID:        strfmt.UUID("054e0100-f50e-4be7-874d-73861179e40d"),
+				inventory:     hostutil.GenerateInventoryWithResourcesWithBytes(1, hardware.GibToBytes(16), "master"),
+				role:          models.HostRoleMaster,
+				srcState:      models.HostStatusDiscovering,
+				dstState:      models.HostStatusInsufficient,
+				statusInfoMsg: "Insufficient CPU to deploy CNV. Required CPU count is: 4 but found: 1",
```

Comment:
What do you think of removing the colons here? I am not sure they are helpful in the sentence.

Reply:
Will remove